### PR TITLE
[INTEL MKL] Fix typo error of an environment variable 

### PR DESCRIPTION
**Pull request by gzmkl**
Change the name of an environment variable
from
     TF_MKL_OPTIMIZE_PRIMITVE_MEMUSE
to
    TF_MKL_OPTIMIZE_PRIMITIVE_MEMUSE

### CODE REVIEW

**Comment by qlzh727** [comment]
Can u fix the cl format lint error?

https://source.cloud.google.com/results/invocations/c3df27fd-ff97-4e5f-b9a6-8f309dcf940b/targets/%2F%2Ftensorflow%2Ftools%2Fci_build:gen_ci_clang_format_out/log

diff --git a/tensorflow/core/util/mkl_util.h b/tensorflow/core/util/mkl_util.h
index 5ea8f2ee47..06c3598f85 100644
--- a/tensorflow/core/util/mkl_util.h
+++ b/tensorflow/core/util/mkl_util.h
@@ -2041,7 +2041,7 @@ class MklPrimitiveFactory {
   static inline bool IsPrimitiveMemOptEnabled() {
     bool is_primitive_mem_opt_enabled = true;
     TF_CHECK_OK(ReadBoolFromEnvVar("TF_MKL_OPTIMIZE_PRIMITIVE_MEMUSE", true,
-          &is_primitive_mem_opt_enabled));
+                                   &is_primitive_mem_opt_enabled));
     return is_primitive_mem_opt_enabled;
   }

**Comment by gzmkl** [comment]
I fixed a couple of coding style issues by running cpplint.py utility. 
But I am not sure if it addressed the issue that Qianli pointed out (not so obvious to me :)).
Thanks!

**Review by tatianashp — CHANGES_REQUESTED** [review]

File: tensorflow/core/util/mkl_util.h [hunk 1]
```diff
@@ -13,8 +13,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 
-#ifndef TENSORFLOW_CORE_UTIL_MKL_UTIL_H_
-#define TENSORFLOW_CORE_UTIL_MKL_UTIL_H_
+#ifndef TENSORFLOW_TENSORFLOW_CORE_UTIL_MKL_UTIL_H_
```
[the remark this answers is not in the document]
Comment (gzmkl) [reply tensorflow/core/util/mkl_util.h]: Well, I ran cpplint.py and somehow the utility complains about it.

If it is fine, let me change it back, to make it consistent across all header files.

**Comment by tatianashp** [comment]
Yes, please change it back to be consistent with the coding style in other
files in this directory.

File: tensorflow/core/util/mkl_util.h (hunk 1 above)
[the remark this answers is not in the document]
Comment (gzmkl) [reply tensorflow/core/util/mkl_util.h]: Done by changing back the macro def.

File: tensorflow/core/util/mkl_util.h [hunk 2]
```diff
@@ -2098,7 +2098,7 @@ static inline memory::format get_desired_format(int channel,
              (channel % 8) == 0) {
     fmt_desired = is_2d
                       ? memory::format::nChw8c
-                      : memory::format::ncdhw;  //not support avx2 for 3d yet.
+                      : memory::format::ncdhw;  // not support avx2 for 3d yet.
```
[the remark this answers is not in the document]
Comment (gzmkl) [reply tensorflow/core/util/mkl_util.h]: Done

**Review by tatianashp — APPROVED** [review]

**Comment by gzmkl** [comment]
Hi Tatiana, 
I noticed that this PR is "merge" blocked due to some CI build issue. 
Please let me know what I need follow up.
Regards,
Guozhong

**Pull request merged**